pages/index.js

```diff
@@ -1,45 +1,52 @@
-import React, { useRef } from 'react';
-import Editor, { Monaco } from '@monaco-editor/react';
+import { Loader } from '@/components/loader';
```

Comment:
the deployment failed, should the package.json include loader dependency as well?  @sohanemon 

<img width="837" alt="Screenshot 2023-08-29 at 14 02 15" src="https://github.com/salesforce-casts/notepad/assets/38701323/b27e6ac6-fff2-4db3-b106-4166aa2c99c5">

Reply:
loader isn't any dependency there. I have created a new component named loader.

Reply:
I don't see any file with the name loader.js and neither do i find the folder components in the branch. am i missing something?